pyveda/vedaset/store/vedabase.py

```diff
@@ -10,6 +10,7 @@
 from pyveda.frameworks.batch_generator import VedaStoreGenerator
 from pyveda import vv
 from pyveda.utils import update_options
+from itertools import zip_longest
```

Comment:
why `zip_longest`, are they potentially different lengths?

Reply:
`itertools` does not support `zip`

Reply:
isn't Py3 `zip` the old `intertools.izip`, like how they swapped out `range` and `xrange`?

Reply:
oh yeah totally you're right I didn't realize that dope

Reply:
I didn't know that either. But they shouldn't be different lengths--that means something went wrong, because images and labels aren't matching up

Reply:
yeah have thought about protecting against such a thing but anyone adding labels or images indiviually should know what they doing anyway, if it comes up down the line can address